docs/astria.md

```diff
@@ -0,0 +1,139 @@
+
+This document outlines what each of the components in the dev-cluster do, and provides instructions for how to run the dev cluster locally.
+
+## Architecture
+At a high level, Astria breaks out all of the components that are historically combined together in monolithic blockchains into individual components. This results in a modular ecosystem that allows users to swap out components to fit their specific needs.
+
+![Astria Architecture](images/astria-architecture.png)
+The diagram above shows a Decentralized Rollup w/ Astria Architecture. Each component is explained in more detail below.
+
+### Users
+One of Astria's goals is to provide rollups with fast finality, atomic cross-rollup inclusion, and decentralization out of the box, without requiring changes to their end user experience. Individual rollups should be able to provide a native experience to their users, Metamask should "just work" with an EVM rollup, and Kepler should "just work" with a cosmos-sdk rollup.
+
+### Rollups
+Astria is designed for permissionless rollup integration and takes advantage of lazy shared sequencing. This means that rollup developers have total sovereignty over their own execution state and can swap out sequencing layers without fear of being locked in. You do not need to ask for permission or go through a governance process to gain access. A rollup simply needs to satisfy the following interfaces:
+1. Write Interface (Rollup <-> Composer):
+    - For transaction submissions.
+2. Read Interface (Rollup <-> Conductor):
+    - For receiving blocks from the shared sequencer network.
+
+The [Composer](#composer) and [Conductor](#conductor) are explained in more detail in future sections.
+
+![Astria Rollup](images/astria-rollups.png)
+Both composer and conductor expose [gRPC](https://grpc.io/) interfaces. See the [astria-protos](https://github.com/astriaorg/astria/tree/main/crates/astria-proto) repo for specific implementation details.
+
+The current dev-cluster deploys a fork of [Geth](https://github.com/astriaorg/go-ethereum) as an EVM rollup. 
+
+### Composer (under construction)
+In the current dev-cluster, the Composer is not yet implemented or included. As of this writing, the Geth rollup mentioned in the previous section simply sends transactions directly to the sequencer network. The Composer will ultimately act as an intermediate component between any given rollup and the sequencer.
+
+The Composer is an in-house abstraction over the potential actors in an MEV market. It contains naive implementations of the roles that a searcher and a builder will fulfill, with the intention of having working examples that economically incentivized third parties can use as a starting point for their own implementations. The abstraction serves the goal of acting as a forcing function on our architecture design by stubbing out the roles and requiring other components to interact with them through an interface.
```

Comment:
The composer does not currently include a builder implementation. Revised paragraph:

```
The Composer is an in-house abstraction over the potential actors in Astria's MEV supply chain. It currently contains a naive implementations of the role a searcher will fulfill, with the intention of having working examples that economically incentivized third parties can use as a starting point for their own implementations. As we progress from design to implementation of Astria's proposer-builder separation it will also include an example implementation of a naive builder. The abstractions serves the goal of acting as a forcing function on our architecture design by stubbing out the roles and requiring other components to interact with them through an interface.
```